src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs

```diff
@@ -422,6 +422,33 @@ public virtual IndexerPageableRequestChain GetSearchRequests(AnimeEpisodeSearchC
             return pageableRequests;
         }
 
+        public virtual IndexerPageableRequestChain GetSearchRequests(AnimeSeasonSearchCriteria searchCriteria)
+        {
+            var pageableRequests = new IndexerPageableRequestChain();
+
+            if (SupportsSeasonSearch)
+            {
+                var queryTitles = TextSearchEngine == "raw" ? searchCriteria.SceneTitles : searchCriteria.CleanSceneTitles;
+                foreach (var queryTitle in queryTitles)
+                {
+                    if (Settings.AnimeStandardFormatSearch && searchCriteria.SeasonNumber > 0)
+                    {
+                        AddTvIdPageableRequests(pageableRequests,
```

Comment:
This needs to be moved outside of the loop, right now it's triggered for each query title instead of just once. Looks like the original search for anime episodes does this as well, I'll fix those up.